frame/system/src/lib.rs

```diff
@@ -1448,7 +1448,7 @@ impl<T: Config> Pallet<T> {
 		EventCount::<T>::kill();
 		<EventTopics<T>>::remove_all();
 	}
-
+	
```

Comment:
seems you have introduced some spaces.

Add a configuration in your editor to remove spaces on empty lines